lib/openstudio-standards/prototypes/common/objects/Prototype.utilities.rb

```diff
@@ -343,13 +343,13 @@ def hspf_to_cop_heating_with_fan(hspf)
   # @param eer [Double] Energy Efficiency Ratio (EER)
   # @param capacity_w [Double] the heating capacity at AHRI rating conditions, in W
   # @return [Double] Coefficient of Performance (COP)
-  def eer_to_cop(eer, capacity_w = nil)
+  def eer_to_cop_no_fan(eer, capacity_w = nil)
```

Comment:
These functions were mislabeled. They did not correspond to a straight EER to COP conversion but rather excluded the effect of the fan when doing the conversion.

Reply:
@khaddad, @ckirney  - as per our discussion/email, see the changes in `lib/openstudio-standards/prototypes/common/objects/Prototype.utilities.rb`.

Reply:
Thanks